### PR TITLE
Version

### PR DESCRIPTION
**Pull request by Ankit-Dhankhar**
This PR is prototype of solution of issue #2592 that I had in mind. Please suggest changes and provide reviews.

### CODE REVIEW

File: syft/version.py [hunk 1]
```diff
@@ -1 +1,206 @@
...
+import distutils
+import locale
+import os
+import platform
+import struct
+import subprocess
+import sys
+import warnings
+
+VERSIONS = {
```
Comment (robert-wagner) [inline syft/version.py]: This all lives in the requirements.txt

File: syft/version.py [hunk 2]
```diff
@@ -1 +1,206 @@
...
+    "lz4": "2.1.6",
+    "numpy": "1.14.0",
+    "sklearn": "0.21.0",
+    "tblib": "1.4.0",
+    "torch": "1.1",
+    "websocket": "0.56.0",
+    "websockets": "7.0",
+}
+
+version_message = (
```
Comment (robert-wagner) [inline syft/version.py]: Pip does all of this automatically (assuming there is an error)

**Comment by Ankit-Dhankhar** [comment]
https://github.com/pandas-dev/pandas/blob/master/.github/ISSUE_TEMPLATE.md
Aim was to make it similar to above.

**Comment by jvmancuso** [comment]
@robert-wagner did you add this to the issue template? Hoping to get a resolution on this

**Comment by iamtrask** [comment]
Closing due to inactivity. This does seem like a really useful contribution though! Please do resubmit!

**Pull request closed without merging**